h/features/test/models_test.py

```diff
@@ -71,3 +73,35 @@ def features_pending_removal_override(self, request):
         }, clear=True)
         patcher.start()
         request.addfinalizer(patcher.stop)
+
+
+class TestCohort(object):
+    def test_init(self):
+        name = "My Hypothesis Cohort"
+
+        cohort = FeatureCohort(name=name)
+        db.Session.add(cohort)
+        db.Session.flush()
+
+        assert cohort.id
+        assert cohort.name == name
+        assert cohort.created
+        assert cohort.updated
+
+    def test_get_by_id_when_id_does_exist(self):
```

Comment:
Makes sense. This leaves no tests left for this model. Should I add any particular ones? Test uniqueness of (user, cohort) pairs?